base_layer/core/src/transactions/transaction_components/encrypted_data.rs

```diff
@@ -90,18 +163,18 @@ pub enum PaymentId {
     U64(u64),
     /// A u256 number.
     U256(U256),
-    /// A Tari address, usually the sender address.
-    Address(TariAddress),
-    /// Open - the user optionally specifies a human-readable string.
-    Open(Vec<u8>),
+    /// Open - the user optionally specifies a human-readable string 'user_data' ('tx_type' is added by the system).
```

Comment:
```suggestion
    /// Open - the user optionally specifies 'user_data' ('tx_type' is added by the system).
```
The idea is to leave it open to interpretation from the wallet, can be anything. Not just a human-readable string.